todo_app/app.py

```diff
@@ -1,34 +1,35 @@
 from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
-from todo_app.data.trello_service import TrelloService
+from todo_app.data.mongo_service import MongoService
```

Comment:
Nice, the fact this is such a simple change in this file is a really good sign that you picked a sensible abstraction level all that time ago 😄